examples/scripts/ct_abel.py

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# This file is part of the SCICO package. Details of the copyright
+# and user license can be found in the 'LICENSE.txt' file distributed
+# with the package.
+
+r"""
+Abel Transform Demo
+===================
+
+This is a placeholder for more detailed docs."
```

Comment:
This needs to be replaced with a description of similar detail to the other example scripts.

Reply:
Fixed.